test/kernel_upgrade.js

```diff
@@ -30,6 +30,14 @@ contract('Kernel Upgrade', accounts => {
         kernelId = await kernel.KERNEL_APP_ID()
     })
 
+    it('checks ERC897 functions', async () => {
+        const kernelProxy = KernelProxy.at(app)
+        const implementation = await kernelProxy.implementation()
+        assert.equal(implementation, kernelBase.address, "App address should match")
+        const proxyType = await kernelProxy.proxyType()
+        assert.equal(proxyType, 2, "Proxy type should be 2 (upgradeable)")
```

Comment:
Rather than using the constant `1` or `2` here and elsewhere, maybe we could use `ERCProxy.FORWARDING()` and `ERCProxy.UPGRADABLE()`?